ignition-server/main.go

```diff
@@ -92,11 +92,11 @@ func NewStartCommand() *cobra.Command {
 	return cmd
 }
 
-// payloadStoreReconciler runs a TokenSecretReconciler controller
+// SetUpPayloadStoreReconciler sets up manager with a TokenSecretReconciler controller
 // to keep the PayloadStore up to date.
-func payloadStoreReconciler(ctx context.Context) error {
+func SetUpPayloadStoreReconciler(ctx context.Context) (ctrl.Manager, error) {
```

Comment:
Doesn't make a difference one way or another as `main` can not be imported, but out of curiosity, why did you export this?

Reply:
I didn't mean to, updated.